localization/i18n/pl/OrcaSlicer_pl.po

```diff
@@ -5439,7 +5440,7 @@ msgid ""
 "Please heat the nozzle to above 170 degree before loading or unloading "
 "filament."
 msgstr ""
-"Przed załadowaniem lub wyładunkiem filamentu, podgrzej dyszę do temperatury "
+"Przed załadowaniem lub rozładunkiem filamentu, podgrzej dyszę do temperatury "
```

Comment:
Similar to above - maybe use "wyjęciem" instead of "rozładunkiem"?

Reply:
j/w